interfaces/seccomp/backend.go

```diff
@@ -82,8 +83,57 @@ func seccompToBpfPath() string {
 // Backend is responsible for maintaining seccomp profiles for snap-confine.
 type Backend struct{}
 
-// Initialize does nothing.
+var globalProfileLE = []byte{
+	0x20, 0x00, 0x00, 0x00, 0x04, 0x00, 0x00, 0x00, 0x15, 0x00, 0x00, 0x04, 0x3e, 0x00, 0x00, 0xc0,
+	0x20, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x35, 0x00, 0x00, 0x01, 0x00, 0x00, 0x00, 0x40,
+	0x15, 0x00, 0x00, 0x0c, 0xff, 0xff, 0xff, 0xff, 0x15, 0x00, 0x06, 0x0b, 0x10, 0x00, 0x00, 0x00,
+	0x15, 0x00, 0x00, 0x02, 0xb7, 0x00, 0x00, 0xc0, 0x20, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+	0x15, 0x00, 0x03, 0x08, 0x1d, 0x00, 0x00, 0x00, 0x15, 0x00, 0x00, 0x07, 0x15, 0x00, 0x00, 0xc0,
+	0x20, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x15, 0x00, 0x00, 0x05, 0x36, 0x00, 0x00, 0x00,
+	0x20, 0x00, 0x00, 0x00, 0x1c, 0x00, 0x00, 0x00, 0x35, 0x00, 0x00, 0x03, 0x01, 0x00, 0x00, 0x00,
+	0x20, 0x00, 0x00, 0x00, 0x18, 0x00, 0x00, 0x00, 0x25, 0x00, 0x00, 0x01, 0x00, 0x00, 0x00, 0x00,
+	0x06, 0x00, 0x00, 0x00, 0x01, 0x00, 0x05, 0x00, 0x06, 0x00, 0x00, 0x00, 0x00, 0x00, 0xff, 0x7f,
+}
+
+var globalProfileBE = []byte{
+	0x00, 0x20, 0x00, 0x00, 0x00, 0x00, 0x00, 0x04, 0x00, 0x15, 0x00, 0x07, 0x80, 0x00, 0x00, 0x16,
+	0x00, 0x20, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x15, 0x00, 0x05, 0x00, 0x00, 0x00, 0x36,
+	0x00, 0x20, 0x00, 0x00, 0x00, 0x00, 0x00, 0x18, 0x00, 0x35, 0x00, 0x03, 0x00, 0x00, 0x00, 0x01,
+	0x00, 0x20, 0x00, 0x00, 0x00, 0x00, 0x00, 0x1c, 0x00, 0x25, 0x00, 0x01, 0x00, 0x00, 0x00, 0x00,
+	0x00, 0x06, 0x00, 0x00, 0x00, 0x05, 0x00, 0x01, 0x00, 0x06, 0x00, 0x00, 0x7f, 0xff, 0x00, 0x00,
+}
+
+func isBigEndian() bool {
+	switch runtime.GOARCH {
+	case "s390x":
+		return true
+	case "ppc":
+		return true
+		// TODO: perhaps more here?
```

Comment:
What did you have in mind? (not a blocker)

Reply:
A better way to detect big-endian vs little-endian architectures. This one feels like a hack.